tests/integration/tier0/non-destructive/config-file/test_config_file.py

```diff
@@ -19,8 +19,16 @@ def remove_files(config):
         os.remove(os.path.expanduser(cfg.path))
 
 
-def test_config_custom_path_custom_filename(convert2rhel):
-    config = [Config("~/.convert2rhel_custom.ini", "[subscription_manager]\nactivation_key = config_activationkey")]
+def test_user_path_custom_filename(convert2rhel):
```

Comment:
You will need to edit also the https://github.com/oamg/convert2rhel/blob/main/tests/integration/tier0/non-destructive/config-file/main.fmf#L17 here. You will need to update the name, tag and pytest call

Reply:
I changed it